Day-I-Social-Card/social-card/src/components/CardComponents/CardContainer.js

```diff
@@ -1,2 +1,26 @@
 import React from 'react';
 import './Card.css';
+import CardBanner from './CardBanner';
+import CardContent from './CardContent';
+
+
+function CardContainer() {
+    function CardLink(e){
+        e.preventDefault
```

Comment:
This isn't doing anything since you aren't invoking it. In order to prevent the default:
```javascript
e.preventDefault();
```
The div doesn't need this, though.